### PR TITLE
Fix Boost build issue

### PR DESCRIPTION
**Pull request by kfli716**
This change is required to build on Linux.

### CODE REVIEW

File: app/src/main/jni/cmake/FindBoost.cmake [hunk 1]
```diff
@@ -1,7 +1,7 @@
 set(Boot_FOUND TRUE)
 
 # libraries that can't compile
-set(BOOST_EXCLUDE_LIBRARIES "context;coroutine;fiber" CACHE STRING "" FORCE)
+set(BOOST_EXCLUDE_LIBRARIES "fiber" CACHE STRING "" FORCE)
```
Comment (graphemecluster) [inline app/src/main/jni/cmake/FindBoost.cmake]: How about also including `fiber` in `BOOST_INSTALLED_LIBRARIES`?

**Comment by graphemecluster** [comment]
And it seems that some other libraries can be excluded, since we don’t use `librime-charcode` and some of them were migrated to C++ built-in libraries

**Comment by graphemecluster** [comment]
And please also include https://github.com/TypeDuck-HK/TypeDuck-Android/commit/875b970fa7aea07f13b4bad87211b2cbe64cdd29#diff-350211d8d47de8bd5c206a11477a8e6228c629ad0491550b5acbd72c9ad55af1 at the least in this PR (just remove them, there’ll be no use commenting them out)

File: app/src/main/jni/cmake/FindBoost.cmake (hunk 1 above)
Comment (kfli716) [reply app/src/main/jni/cmake/FindBoost.cmake]: I can build with fiber included, and I just changed the RimePlugins.cmake as you suggested.

**Pull request merged**